src/extractors/getFixedTFunction.ts

```diff
@@ -0,0 +1,99 @@
+import * as BabelTypes from '@babel/types';
+import * as BabelCore from '@babel/core';
+import extractTFunction from './tFunction';
+import { ExtractedKey } from '../keys';
+import { Config } from '../config';
+import { getFirstOrNull, evaluateIfConfident } from './commons';
+import { CommentHint, getCommentHintForPath } from '../comments';
+
+/**
+ * Check whether a given CallExpression path is a call to `useTranslation` hook.
+ * @param path: node path to check
+ * @param config: plugin configuration
+ * @returns true if the given call expression is indeed a call to
+ *   `useTranslation`
+ */
+function isGetFixedTFunction(
+  path: BabelCore.NodePath<BabelTypes.CallExpression>,
+  config: Config,
+): boolean {
+  const callee = path.get('callee');
+
+  if (!callee.isMemberExpression()) return false;
+
+  const obj = callee.get('object');
+  if (!obj.isIdentifier()) return false;
+
+  const prop = callee.get('property');
+  if (Array.isArray(prop) || !prop.isIdentifier()) return false;
+
+  return (
+    config.i18nextInstanceNames.includes(obj.node.name) &&
+    prop.node.name === 'getFixedT'
+  );
+}
+
+/**
+ * Parse `useTranslation()` hook to extract all its translation keys and
```

Comment:
Wrong docstring. Could you add a link to the official i18next documentation while you're at it? I wasn't aware that getFixedTFunction was a thing.